SimpleTextingDotNet/v2/Model/Object/Contact.cs

```diff
@@ -0,0 +1,105 @@
+﻿// <copyright>
+// MIT License
+// 
+// Copyright (c) 2022 Kingdom First Solutions
+// 
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+// 
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+// 
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using static SimpleTextingDotNet.Enum;
+
+namespace SimpleTextingDotNet.v2.Model.Object
+{
+    /// <summary>
+    /// The contact data.
+    /// </summary>
+    public class Contact
+    {
+        /// <summary>
+        /// Gets or sets the contact identifier in hexadecimal format.
+        /// </summary>
+        public string ContactId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Contact Phone.
+        /// </summary>
+        public string ContactPhone { get; set; }
+
+        /// <summary>
+        /// Gets or sets the First Name.
+        /// </summary>
+        public string FirstName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Last Name.
+        /// </summary>
+        public string LastName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Email
+        /// </summary>
+        public string Email { get; set; }
+
+        /// <summary>
+        /// Contact's birthday in format (yyyy-mm-dd)
+        /// </summary>
+        public DateTime Birthday { get; set; }
```

Comment:
Test nullable datetime.
```suggestion
        public DateTime? Birthday { get; set; }
```

Reply:
This was indeed the issue, RestSharp was "handling" it when it came through as null, fixed now.